include/openthread/types.h

```diff
@@ -812,18 +812,8 @@ typedef enum otRoutePreference
     OT_ROUTE_PREFERENCE_HIGH = 1,   ///< High route preference.
 } otRoutePreference;
 
-enum
-{
-    /**
-     * Maximum size of Service Data in bytes.
-     */
-    kMaxServiceDataSize = 252,
-
-    /**
-     * Maximum size of Server Data in bytes. This is theoretical limit, practical one is much lower.
-     */
-    kMaxServerDataSize = 248,
-};
+#define OT_MAX_SERVICE_DATA_SIZE 252    ///< Maximum size of Service Data in bytes.
+#define OT_MAX_SERVER_DATA_SIZE 248     ///< Maximum size of Server Data in bytes. This is theoretical limit, practical one is much lower.
```

Comment:
Couple nits:
1. renaming to `OT_SERVICE_DATA_MAX_SIZE` would better follow convention in this file.
1. prefer to align values.